File: Problems_and_Solutions/days-10 to 19/day-11/Java/Solution3.java
class Node {
    int data;
    Node next;
    Node(int data) {
        this.data = data;
        this.next = null;
    }
}

class Solution3 {
    public static boolean searchKey(int n, Node head, int key) {
        Node temp = head;
        while(temp != null){
            if(temp.data == key){
                return true;
            }
            temp = temp.next;
        }
        return false;
    }
    public static void main(String[] args) {
        Node head = new Node(1);
        head.next = new Node(2);
        head.next.next = new Node(3);
        int n = 3;
        int key = 2;
        System.out.println(searchKey(n,head,key));
    }
}

File: Problems_and_Solutions/days-10 to 19/day-11/cpp/solution 3.cpp
class Solution {
  public:
    // Function to count nodes of a linked list.
    bool searchKey(int n, Node* head, int key) {
        // Code here
        while(head!=NULL)
        {
            if(head->data==key)
                return true;
            head=head->next;
        }
        return false;
    }
};